doctest/test_docs.py

```diff
@@ -282,7 +299,7 @@ def set_up_test_indices_without_calcite(test):
 
 
 def tear_down(test):
-    get_test_data_manager().cleanup_indices()
+    pass
```

Comment:
Thanks for the PR I was thinking about improving this as well. Have a question, will this lead to cascading test failures if the first test failed to load indexes?